### ISSUE TITLE
Open recent fsproj file results in window hanging

### ISSUE THREAD

**Issue by slav** [Microsoft/vscode#11983]
- VSCode Version: 1.5.2
- OS Version: Windows 10 Anniversary edition

Steps to Reproduce:

1. Create F# project in visual studio 2015
1.5... 
2. Open source folder in visual studio code
3. Open fsproj file
4. Restart VS code
5. On restart when fsproj is loaded into editor the VS window hangs. File, Edit, View, Go, Help menu still works. Cursor changes to hand above the hanging window, but I'm able to open new window through File -> New Window.

p.s. This doesn't happen on the flesh fsproj file created by Visual Studio 2015. But when I opened with a project I've worked for a little bit, it hangs.

Any advise on how to fix this or diagnose what's causing window to freeze?

**Comment by bpasero** [comment]
I wonder if this issue is being caused by an installed extension. Can you try to run VS Code without extensions? From the command line, execute: `code --disable-extensions` and try your steps again to see if it reproduces. If you see it is an issue with the extension, please file it against the extension repository itself.

**Comment by slav** [comment]
No, still hanging. I thought extensions might be the cause so uninstalled them all manually.

From the log in console when I clicked close window.
```
C:\Program Files (x86)\Microsoft VS Code>code --disable-extensions

C:\Program Files (x86)\Microsoft VS Code>
[15508:9332:0915/190238:VERBOSE1:crash_service_main.cc(68)] Session start. cmdline is [--reporter-url=https://ticinocrashreporter.azurewebsites.net/crash --application-name=VSCode --v=1]
[15508:9332:0915/190238:VERBOSE1:crash_service.cc(142)] window handle is 0005151E
[15508:9332:0915/190238:VERBOSE1:crash_service.cc(290)] pipe name is \\.\pipe\VSCode Crash Service
dumps at C:\Users\user\AppData\Local\Temp\VSCode Crashes
[15508:9332:0915/190238:VERBOSE1:crash_service.cc(294)] checkpoint is C:\Users\user\AppData\Local\Temp\VSCode Crashes\crash_checkpoint.txt
server is https://ticinocrashreporter.azurewebsites.net/crash
maximum 128 reports/day
reporter is electron-crash-service
[15508:9332:0915/190238:VERBOSE1:crash_service_main.cc(84)] Ready to process crash requests
[15508:14544:0915/190238:VERBOSE1:crash_service.cc(323)] client start. pid = 19512
[15508:14544:0915/190238:VERBOSE1:crash_service.cc(323)] client start. pid = 20032
[12224:12368:0915/190238:VERBOSE1:crash_service_main.cc(68)] Session start. cmdline is [--reporter-url=https://ticinocrashreporter.azurewebsites.net/crash --application-name=VSCode --v=1]
[12224:12368:0915/190238:VERBOSE1:crash_service.cc(142)] window handle is 00160D6E
[12224:12368:0915/190238:VERBOSE1:crash_service.cc(290)] pipe name is \\.\pipe\VSCode Crash Service
dumps at C:\Users\user\AppData\Local\Temp\VSCode Crashes
[12224:12368:0915/190238:VERBOSE1:crash_service.cc(294)] checkpoint is C:\Users\user\AppData\Local\Temp\VSCode Crashes\crash_checkpoint.txt
server is https://ticinocrashreporter.azurewebsites.net/crash
maximum 128 reports/day
reporter is electron-crash-service
[12224:12368:0915/190238:ERROR:crash_service.cc(301)] could not start dumper
[15508:18924:0915/190250:VERBOSE1:crash_service.cc(331)] client end. pid = 19512
[15508:14544:0915/190252:VERBOSE1:crash_service.cc(323)] client start. pid = 21536
[15508:14544:0915/190252:VERBOSE1:crash_service.cc(323)] client start. pid = 20032
[20656:2860:0915/190252:VERBOSE1:crash_service_main.cc(68)] Session start. cmdline is [--reporter-url=https://ticinocrashreporter.azurewebsites.net/crash --application-name=VSCode --v=1]
[20656:2860:0915/190252:VERBOSE1:crash_service.cc(142)] window handle is 001E1122
[20656:2860:0915/190252:VERBOSE1:crash_service.cc(290)] pipe name is \\.\pipe\VSCode Crash Service
dumps at C:\Users\user\AppData\Local\Temp\VSCode Crashes
[20656:2860:0915/190252:VERBOSE1:crash_service.cc(294)] checkpoint is C:\Users\user\AppData\Local\Temp\VSCode Crashes\crash_checkpoint.txt
server is https://ticinocrashreporter.azurewebsites.net/crash
maximum 128 reports/day
reporter is electron-crash-service
[20656:2860:0915/190252:ERROR:crash_service.cc(301)] could not start dumper
[9712:5064:0915/190252:VERBOSE1:crash_service_main.cc(68)] Session start. cmdline is [--reporter-url=https://ticinocrashreporter.azurewebsites.net/crash --application-name=VSCode --v=1]
[9712:5064:0915/190252:VERBOSE1:crash_service.cc(142)] window handle is 001814E2
[9712:5064:0915/190252:VERBOSE1:crash_service.cc(290)] pipe name is \\.\pipe\VSCode Crash Service
dumps at C:\Users\user\AppData\Local\Temp\VSCode Crashes
[9712:5064:0915/190252:VERBOSE1:crash_service.cc(294)] checkpoint is C:\Users\user\AppData\Local\Temp\VSCode Crashes\crash_checkpoint.txt
server is https://ticinocrashreporter.azurewebsites.net/crash
maximum 128 reports/day
reporter is electron-crash-service
[9712:5064:0915/190252:ERROR:crash_service.cc(301)] could not start dumper
[VS Code]: detected unresponsive
[15508:17908:0915/190328:VERBOSE1:crash_service.cc(331)] client end. pid = 21536
[15508:17908:0915/190328:VERBOSE1:crash_service.cc(331)] client end. pid = 20032
[15508:18924:0915/190328:VERBOSE1:crash_service.cc(331)] client end. pid = 20032
[15508:18924:0915/190329:VERBOSE1:crash_service.cc(352)] zero clients. exiting
[15508:9332:0915/190329:VERBOSE1:crash_service.cc(490)] session ending..
[15508:9332:0915/190329:VERBOSE1:crash_service.cc(495)] clients connected :4
clients terminated :4
dumps serviced :0
dumps reported :0
[15508:9332:0915/190329:VERBOSE1:crash_service_main.cc(89)] Session end. return code is 0
```

**Comment by bpasero** [comment]
@slav when it hangs, does it eventually load or just shows an empty window? When it hangs can you try to open the developer console (Help > Toggle Developer Tools) and see if any output is printed?

**Comment by bpasero** [comment]
A number of days has past without answer on the questions asked. I am thereby going to close this issue to clean up our backlog of issues. Once you can answer on the questions, the issue can be reopened again. Thanks for understanding this process!

**Comment by slav** [comment]
@bpasero - It shown explorer and the tree of files in projects, fsproj file with text in it and left sidebar with icons to switch to debugger, extensions, etc. At that point it freezes. I've opened with developer tools opened.

Console is effectively blank. Elements is Blank. I can open Sources. I tried to pause script execution in debugger, but nothing happened.

The screen just hangs as is, all files opened, drawing working (I can resize etc.), but I can't interact with it. Also, I was playing with console buttons and clicked on toggle device mode. The screen somewhat collapsed and isn't expanding again. I can resize window fine and it's being drawn, just content not expanded back out.
![screenshot_3](https://cloud.githubusercontent.com/assets/350987/18693243/4eb1e788-7f6e-11e6-9896-13710bd58110.png)
